service/src/main/java/bio/terra/workspace/app/controller/ReferencedGcpResourceController.java

```diff
@@ -78,7 +78,6 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
                 CloningInstructions.fromApiModel(body.getMetadata().getCloningInstructions()))
             .bucketName(body.getBucket().getBucketName())
             .build();
-    resource.validate();
```

Comment:
Why are these not necessary?

Reply:
Every subclass of `WsmResource` calls `validate` from every constructor, and since these are immutable objects this lets us enforce validity at object creation time. This call is doing duplicate validation with the `build()` call above